<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AES File Encryption/Decryption</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="encryptButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileSelectorButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File to Decrypt/Encrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileSelectTextLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Console Log:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="operationModeComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ECB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CBC</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="fileSelectTextLabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode of operation</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="keyInputField">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>351</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="keyLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ivInputField">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>351</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ivLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialization vector (iV)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="statusMessageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>741</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="outputFileNameLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file name</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="writeFileNameInput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
